braintree/resource.py

```diff
@@ -17,7 +17,7 @@ def verify_keys(params, signature):
 
     @staticmethod
     def __flattened_params_keys(params, parent=None):
-        if isinstance(params, str):
+        if isinstance(params, str) or isinstance(params, unicode):
```

Comment:
We have a fix for this that will be deployed with the next release that will handle python 2 and 3's `str` as well as `unicode` and `bytes`.